### ISSUE TITLE
Cable not connecting / Invisible after relog (1.18 Fabric , Multiplayer )

### ISSUE THREAD

**Issue by Marci1125** [AppliedEnergistics/Applied-Energistics-2#5972]
### Describe the bug

If you place any kind of cable it's visual not connected but it works. After a relog the builded cable is shown invisible but you can't pass the airblock. You can't mine the invisible cable to get it back. To delete the corrupted air block you need to place a new cable at the same point and destroy it.

Other Players can't see if you placing cable, that is generally invisibl
(Placing some cables)
![2022-01-06_17 40 55](https://user-images.githubusercontent.com/97245287/148418052-0c6ea5fb-0300-4a66-ad5d-17bd5a9acdf2.png)

After Relog its invisible
![2022-01-06_17 41 14](https://user-images.githubusercontent.com/97245287/148418179-82037057-f5d6-42da-92ef-ba25afbc2945.png)

### How to reproduce the bug

Just place something and relog

### Expected behavior

Cable connection visuable and visible after relogg

![2022-01-06_17 48 37](https://user-images.githubusercontent.com/97245287/148419066-6df749cb-c717-4967-b10c-c690747db9ce.png)

### Additional details

In the Singleplayer its all workin fine, maybe a server config thing?

### Which minecraft version are you using?

1.18

### On which mod loaders does it happen?

Fabric

### Crash log

https://gist.github.com/Marci1125/70aad14d71ee249fad3a07f3ec5d5b29

**Comment by shartte** [comment]
@Marci1125 Is your pack available somewhere to download easily?

**Comment by Marci1125** [comment]
@shartte it's not a modpack, the mods on the server are just singlemods , you can download our used mods under this link https://we.tl/t-qWPDdLicnJ (The link expires in 7 days from this time stamp)

After various methods i tried , i guess it's maybe incompatible with the server. The Server is hosted by Shockbyte. I run a private server on my computer and everything is working fine with the mod... Singeplayer all fine too. 

Since we're not editing any config files , just throw the jar in the mod folder as usual nothing should be changed there. i read the configs and can't find anything like "hey if you wrote this from true to false the whole mod doesn't work" ... 

 Well, the mod is working on a self hosted server... i have no idea where to search for the issue :/ 

From Google Search and the Dev Discord from AE2 i get the answers to deinstall everything with chunkloaders and everything with optifine. Since we havn't such mods this not helped us. 

We'll thank you for any idea to fix the problem :-)

**Comment by shartte** [comment]
@Marci1125 I tested the mods locally and couldn't actually repro it :-|

Are you still having these issues with the 10.0.0 release?

**Comment by Marci1125** [comment]
yes issue is still alive, the 10.0.0 release didn't change anything... I think the mod is incompatible with Shockbyte as a Server Hoster... Sadly but i gave up the mod for our multiplayer world.

**Comment by shartte** [comment]
If you have it still installed, could you please try a few things with a debug card?

First of all: `/give @s ae2:debug_card`

Then click on one of the disconnected cables with the debug card in your hand.
This should print a bunch of stuff to your server log. Please paste that here.

It should say something along the lines of:

```
-- Grid Details
Grid Powered: false
Grid Booted: true
[...]
-- Delayed Init Details
QueuedForReady: 1
ReadyInvoked: 1
```

**Comment by Marci1125** [comment]
Grid Details
Grid Powered: true
Grid Booted: true
Nodes in grid: 38
Grid Pivot node: node hosted by 
appeng.parts.networking.GlassCablePart @ 6343,86,1963
[...]
This Node: node hosted by 
appeng.parts.networking.GlassCablePart @ 6343,86,1963
This Node Active: true
Node exposed on side south
Cable distance: 7
--CableBus Details
in World:true
Has Redstone: false
Node Channels: 1
Channels down: 0
Channels up: 0
Channels west: 1
Connected Sides DUnsWe
## Delayed Init Details
QueuedForRelay: 1
ReadyInvoked: 1

https://imgur.com/a/gvexbYW

**Comment by shartte** [comment]
Okay the debug card says that the cable is working normally on the server-side.

The crash report you provided is from your client, right? You don't have access to the actual server?
Because you might be right. This might be something your server-host is doing.

At this point, it really seems like the server->client synchronization packet is not arriving or is being modified in flight.

The only other thing I can think of: Can you post your server and client log somewhere? Preferrably right as you are logging in/placing cables.
You can DM me on discord, if you don't want to post it publicly.

**Comment by shartte** [comment]
Debugged -> Turned out to be a mismatch of Block Entity Type IDs between server and client. This led to any server->client data for AE2 block entities to be discarded.

Fixed by updating from Fabric 0.44 -> 0.45+ which introduces block entity type synchronization between server and client.
